environment.yml

```diff
@@ -11,3 +11,5 @@ dependencies:
     - pandas>=1.5
     - matplotlib>=3.6
     - seaborn>=0.12
+    - ipykernel>=6.20
```

Comment:
Is this needed separately from the `ipykernel>=6.0` on line 7? Or should that original reference be updated?

Reply:
Okay, I checked again and made sure to completely close and reopen VSCode every time I made changes to the environment. `notebook` definitely is needed (although I have no idea what it is), but the duplicate `ipkernel` isn't (obviously).

I now just feel like there's something confusing and/or janky about VSCode's interaction with conda environments, but I can figure that out another time.